trestle/transforms/utils/osco_helper.py

```diff
@@ -0,0 +1,444 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2021 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Facilitate OpenShift Compliance Operator report to NIST OSCAL json transformation."""
+
+import base64
+import bz2
+import logging
+import uuid
+from typing import Any, Dict, List, ValuesView
+from xml.etree import ElementTree
```

Comment:
See this: https://github.com/IBM/compliance-trestle/issues/535 issue. Not a blocker, however, should be addressed before prod usage.

Reply:
OK, revised to use defusedxml for parse. Continue to use xml.etree.ElementTree.Element for signature typing, which I believe is correct.